green_cli/common.py

```diff
@@ -113,14 +113,23 @@ def getsystemmessages(session):
 
 @green.command()
 @with_login
-def listen(session):
+@click.option('--unique', type=bool, default=False)
+@click.option('--ignore', type=str)
+def listen(session, unique, ignore):
     """Listen for notifications.
 
     Wait indefinitely for notifications from the gdk and print then to the console. ctrl-c to stop.
     """
+    ignore = [s.strip() for s in ignore.split(',')] if ignore else []
+    last = None
     while True:
         try:
-            click.echo(format_output(session.notifications.get(block=True, timeout=1)))
+            n = session.notifications.get(block=True, timeout=1)
+            if n == last or n.get('event', None) in ignore:
```

Comment:
Did you mean to say `if n == last and unique`?

Reply:
I broke unique when adding ignore, updated now, thanks.